samples/net/wpan_serial/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _wpanusb_serial-sample:
+
+Application implementing 802.15.4 "serial-radio" protocol
+#########################################################
+
+How to use
+**********
+
+#. Build and flash Zephyr application to the board with 802.15.4 radio.
```

Comment:
#. Build and flash the sample Zephyr application to a board
       with a 802.15.4 radio (such as XXX)